api.planx.uk/modules/pay/service/inviteToPay/createPaymentSendEvents.ts

```diff
@@ -66,6 +66,17 @@ const createPaymentSendEvents = async (
         comment: `bops_submission_${payload.sessionId}`,
       });
       combinedResponse[Destination.BOPS] = bopsEvent;
+
+      const isProduction = process.env.APP_ENVIRONMENT === "production";
+      if (!isProduction) {
+        const bopsV2Event = await createScheduledEvent({
+          webhook: `{{HASURA_PLANX_API_URL}}/bops-v2/${teamSlug}`,
+          schedule_at: new Date(now.getTime() + 30 * 1000),
+          payload: eventPayload,
+          comment: `bops_v2_submission_${payload.sessionId}`,
+        });
+        combinedResponse["bops_v2"] = bopsV2Event;
+      }
```

Comment:
I'm only creating the v2 events on non-production environments to match the current BOPS API functionality, this will eliminate noise of expected failures !

Reply:
Nice simple solution 👌